app/shared/containers/Global/Account/Import/Elements/AccountList.js

```diff
@@ -27,8 +28,14 @@ class GlobalModalAccountImportElementsAccountList extends Component<Props> {
       if (data) {
         const authorizationList = new EOSAccount(data).getAuthorizations(publicKey);
         authorizationList.forEach((authorization) => {
-          authorizations[account] = authorizations[account] || [];
-          authorizations[account].push(authorization);
+          const existingWallet = wallets.find(wallet => {
+            return authorization.perm_name === wallet.authorization &&
+              account === wallet.account;
```

Comment:
This is going to need to compare chainId as well - otherwise it'll trigger if you have the same account/permission on another chain.

Reply:
👍 nice catch!